### ISSUE TITLE
Release 2.8.1 breaks existing configuration/documentation

### ISSUE THREAD

**Issue by maxkratz** [almir/docker-webhook#28]
Version `2.8.0` is able to run (e.g., with no hook config) like described in the README.md:
```
$ docker run --rm -p 9000:9000 -v --name=webhook \
  almir/webhook:2.8.0 -verbose -hooks=/etc/webhook/hooks.json -hotreload
```
Output:
```
[webhook] 2023/06/24 07:58:01 version 2.8.0 starting
[webhook] 2023/06/24 07:58:01 setting up os signal watcher
[webhook] 2023/06/24 07:58:01 attempting to load hooks from /etc/webhook/hooks.json
[webhook] 2023/06/24 07:58:01 couldn't load hooks from file! open /etc/webhook/hooks.json: no such file or directory
[webhook] 2023/06/24 07:58:01 os signal watcher ready
[webhook] 2023/06/24 07:58:01 serving hooks on http://0.0.0.0:9000/hooks/{id}
```

Of course, this also works when passing a valid `hooks.json` file via a Docker volume, e.g., when ran on the CLI or via `docker compose`.

However, once I change the version to the latest release `2.8.1`, the container is unable to start:
```
$ docker run --rm -p 9000:9000 -v --name=webhook \
  almir/webhook:2.8.1 -verbose -hooks=/etc/webhook/hooks.json -hotreload
```
Output:
```
exec /usr/local/bin/webhook: no such file or directory
```

It seems to me that the binary is missing. Running it with a proper passed `hooks.json` file does not change this behavior.

**Comment by almir** [comment]
Thanks for reporting. This should be fixed with 6d39f9b.

**Comment by maxkratz** [comment]
> Thanks for reporting. This should be fixed with [6d39f9b](https://github.com/almir/docker-webhook/commit/6d39f9b740332983c330f5e16c84ec77bfe062fc).

Thank you for the quick fix.
